docs/UserGuide.md

```diff
@@ -3,271 +3,443 @@ layout: page
+**Purpose**: Edits an existing client contact in the ClientBook.
 
-Format: `list [-ATTRIBUTE]…​`
+**Format**: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/POLICY_NUMBER] [t/TAG]…​`
 
-Examples:
-* `list` Shows a list of all clients and all their information
-* `list -policy` Shows a list of all clients and their policy numbers
-* `list -phone -policy` Shows a list of all clients, their phone number, and their policy numbers
+* Edits the client at the specified `INDEX`.
+    * The index refers to the index number shown in the displayed client list.
+    * The index must be a positive integer 1, 2, 3, …​
+* At least one of the optional fields must be provided.
 
-### Editing a person : `edit`
+:exclamation: **Caution**: Existing values will be **replaced** with the input values.
 
-Edits an existing client in the ClientBook.
+**Examples**:
+*  Edit the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
+    * `edit 1 p/91234567 e/johndoe@example.com`
+      
+      ![edit 1](images/edit-1.png)
+    
+*  Edit the name of the 2nd person to be `Betsy Crower`.
+    * `edit 2 n/Betsy Crower`
+    
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/POLICY_NUMBER] [t/TAG]…​`
+### List all clients : `list`
 
-* Edits the client at the specified INDEX. The index refers to the index number shown in the displayed client list. The index must be a positive integer 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
+**Purpose**: Shows a list of all clients in ClientBook. An optional attribute option can be added to show a list with only the specified attribute.
 
+**Format**: `list [-ATTRIBUTE]`
 
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower` Edits the name of the 2nd person to be `Betsy Crower`.
+**Examples**: 
+* Shows a list of all clients and all their information.
+  * `list`
+    
+    ![list](images/list.png)
+    
+*  Shows a list of all clients and their policy number.
+   * `list -policy`
 
-### Locating persons by name: `find`
+[comment]: <> (  insert screenshot)
+*  Shows a list of all clients and their phone number.
+   * `list -phone -policy`
+    
+     ![list phone](images/list-phone-policy.png)
 
-Finds and displays all clients whose field (name, phone, email, address, tags, insurance policy) contains any of the given keywords.
 
-![find_alex_david](images/find_alex_david.png)
+  
+### Search for client contact based on keywords: `find`
 
-![find_alex_&_david](images/find_alex_&_david.png)
+**Purpose**: Finds and displays all clients whose field (name, phone, email, address, tags, insurance policy) contains any of the given keywords.
 
 ![find_alex_&_david_with_filter](images/find_alex_&_david_with_filter.png)
```

Comment:
The image format is different from the other images (e.g. the title bar not shown).

Reply:
Okay, added!